### PR TITLE
feat: add dig command

### PR DESCRIPTION
**Pull request by tom139**
## Description

This command perform a `dig` with the specified name and an optional second argument (usually the type).

## Type of change

- [x] New script command

## Screenshot

image

## Dependencies / Requirements

This step depends solely on the _dig_ command, which comes with the standard developer tools. I think this dependency is expected and unavoidable.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/dig.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# @raycast.title Dig
+# @raycast.mode fullOutput
+#
+# Optional parameters:
+# @raycast.icon 🌍
+# @raycast.packageName Internet
+# @raycast.argument1 { "type": "text", "placeholder": "name" }
+# @raycast.argument2 { "type": "text", "placeholder": "type", "optional": true }
+#
+# @raycast.description "Perform a dig command with the specified options"
```
Comment (unnamedd) [inline commands/developer-utils/dig.sh:16]:
```suggestion
# @raycast.description Perform a dig command with the specified options
```

File: commands/developer-utils/dig.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# @raycast.author Tommaso Panozzo
+# @raycase.authorURL "https://github.com/tom139"
```
Comment (unnamedd) [inline commands/developer-utils/dig.sh:18]:
```suggestion
# @raycase.authorURL https://github.com/tom139
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
I made two simple suggestions, which you can solve right here without need to  commit anything locally.

Thanks for your PR, @tom139. 🙂

**Comment by tom139** [comment]
Thank you for how quickly you responded to this PR, and thank you for the great product you are creating.
I really appreciate all the effort you are putting in creating an active community around Raycast. 💪🏽

**Review by tom139 — COMMENTED** [review]
I addressed changes, hope everything is fine for merging, otherwise let me know. :)

**Pull request merged**